FirebaseRemoteConfigSwift/Sources/PropertyWrapper/RemoteConfigValueObservable.swift

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+import FirebaseRemoteConfig
+
+@available(iOS 14.0, macOS 11.0, macCatalyst 14.0, tvOS 14.0, watchOS 7.0, *)
+internal class RemoteConfigValueObservable<T: Decodable>: ObservableObject {
+  @Published var configValue: T
+  private let key: String
+  private let remoteConfig: RemoteConfig
+  private var registration: FIRConfigUpdateListenerRegistration?
```

Comment:
Random comment: this sticks out as there shouldn't be a `FIR` prefix. Should be fixed in the ObjC source.

Reply:
@karenyz @qdpham13 We should adjust the realtime API on this

Reply:
The `FIR` prefix should never be used in Swift APIs, and always used in Objective-C. Yep, just dropping the FIR prefix for Swift works.

Reply:
@karenyz let me know if you want to include this to the RC API review.